api-reference/_hidden/dxDataGridColumn/cellTemplate.md

```diff
@@ -58,9 +58,13 @@ A function that returns the variable that should be tracked.
 A function called when this variable changes.
 
 ---
-[note] If you implement two-way data binding in your template, set [twoWayBindingEnabled](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/twoWayBindingEnabled.md '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#twoWayBindingEnabled') to **false** to switch off the built-in implementation of this feature.
+The following specificities should be taken into account when you use a **cellTemplate**:
 
-[note] In [fixed columns](/api-reference/10%20UI%20Widgets/GridBase/1%20Configuration/columnFixing '{basewidgetpath}/Configuration/columnFixing/'), a **cellTemplate** is initialized and rendered twice for each cell.
+- When the **DataGrid** is [exported](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/export/), it omits customizations made in the template. However, you can recreate them in the exported file using the ExcelJS API. Use the [customizeCell](/Documentation/ApiReference/Common/Object_Structures/ExportDataGridProps/#customizeCell) function to do this. [View Demo](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/ExcelJSCellCustomization/)
```

Comment:
What is the reason for adding a demo link as in this paragraph instead of the demo button?

Reply:
Demo buttons in a list look ugly.